### DISCUSSION TITLE
Consider removing Google Analytics code

### DISCUSSION THREAD

**Discussion by rawnsley** [mozilla/hubs#3636] [General Questions & Discussion] [answered]
Mozilla Hubs and Hubs Cloud both include Google Analytics code in multiple places with the following template:

```
<script>
if (navigator.doNotTrack !== "1") {
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
}
</script>

```
I would consider dropping Google Analytics completely and replacing it with your own tracking system that doesn't expose data to third parties.

Also `doNotTrack` [is considered obsolete](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack#Specifications) and Safari throws up this warning:

<img width="1469" alt="Screenshot 2021-01-04 at 09 04 51" src="https://user-images.githubusercontent.com/303516/103519509-8bdf4880-4e6d-11eb-83f2-179fabb4b51d.png">

**Comment by robinkwilson** [comment, marked as answer]
We definitely appreciate the concern, thanks for making the discussion thread!

I'd definitely recommend creating a feature request/issue in [Hubs issues](https://github.com/mozilla/hubs/issues). It will be looked over by our team! 

Although this relates indirectly to your question: we err on collecting less data, you can read about it here on the Mozilla blog: [Your Privacy and Mozilla Hubs](https://blog.mozvr.com/your-privacy-and-mozilla-hubs/)
